hashicorp/boundary-ui
# chore: Migrate `form.input` to use `Hds` text input field

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12478

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12478)

## Description
- Migrate `cluster-url.hbs` to use `Hds::Form::TextInput::Field` instead of  `form.input`.



## Screenshots (if appropriate):
###Before: 
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/ab0c3211-79b4-41a6-948b-e16a474df547)

###After: 
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/57091633-4287-4a9c-8db2-1eb11f8b6a72)

## How to Test
- Start desktop
- Ensure that `serve://boundary` is automatically filled and autofocus goes to input field on startup





:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:


~~- [ ] I have added JSON response output for API changes~~
~~- [ ] I have added steps to reproduce and test for bug fixes in the description~~
~~- [ ] I have commented on my code, particularly in hard-to-understand areas~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
